index.js

```diff
@@ -1,17 +1,60 @@
 class SortedList {
-  constructor() {}
+  constructor(items, length) {
+    this.items = [],
+    this.length =this.items.length
+  }
 
-  add(item) {}
+  add(item) {
+    const list = this.items.push(item)
+    list.sort(function(a, b) {
+      return a - b
+    })
+    this.length = list.length
```

Comment:
Y aqui es quitar el list por this.items
```suggestion
    this.length = this.items.length
```